tables/ref_isolate_pairs/riou21-pair.csv

```diff
@@ -0,0 +1,3 @@
+﻿ref_name,control_iso_name,iso_name
+Riou21,S:614G,B.1.351 Spike:-246I-614G
+Riou21,B.1.351 Spike:-246I-614G,S:614G
```

Comment:
This is correct, you should add this row. The CP_Wave2 was about Beta variant infection. The database check will make sure the infection matchs the control_iso_name. That's why the program throws error when this row is missing.